pyspark/test/bigdl/keras1/test_layer.py

```diff
@@ -118,6 +165,11 @@ def test_maxpooling1d(self):
                              input_shape=(3, 20),)
         self.modelTestSingleLayer(input_data, layer)
 
+    def test_globalmaxpooling3d(self):
+        input_data = np.random.random_sample([1, 5, 20, 25, 35])
+        layer = GlobalMaxPooling3D(dim_ordering="th", input_shape=(5, 20, 25, 35))
```

Comment:
what if we don't specify `dim_ordering` here?

Reply:
For 3D layers, seems we only support `th`. Thus, we must specify.